### DISCUSSION TITLE
Custom name for 'Introduction' page

### DISCUSSION THREAD

**Discussion by dmitriyrotaenko** [PaloAltoNetworks/docusaurus-openapi-docs#1057] [Q&A] [answered]
Hi, is there a way to provide a custom name for the **Introduction** page generated from the `info` block? So, instead of **Introduction**, I would have a page called **Gobbledygook** generated from the `info` block.

**Comment by raldred** [comment, marked as answer]
If I have understood you correctly, you might be able to use:
`x-displayName: Gobbledygook` in side your `info` block

This will change the label in the sidebar but the page URL will still be whatever your `info` -> `title` is

**Reply by andrekutianski** [reply]
This not worked for me
